app/conf/app.conf

```diff
@@ -615,9 +615,9 @@ form_element_display_format_without_label = <div class='formLabel'>^ELEMENT</div
 form_element_error_display_format = <div class='formLabel'>^EXTRA^LABEL (<span class='formLabelError'>^ERRORS</span>)<br/>^ELEMENT</div>
 
 # Used for bundle-able fields such as attributes
-bundle_element_display_format = <div class='bundleLabel'>^LABEL ^DOCUMENTATIONLINK ^ELEMENT</div>
+bundle_element_display_format = <div class='bundleLabel ^CSSCLASSES'>^LABEL ^DOCUMENTATIONLINK ^ELEMENT</div>
```

Comment:
This creates a redundant whitespace character when there are no classes, but is consistent with the other elements on the same line (e.g. between `^LABEL` and `^DOCUMENTATIONLINK`).